addons/website_slides/static/tests/tours/slides_course_reviews_comment.js

```diff
@@ -0,0 +1,48 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("course_reviews_comment", {
+    url: "/slides",
+    steps: () => [
+        {
+            trigger: "a:contains(Basics of Gardening - Test)",
+            run: "click",
+        },
+        {
+            trigger: "a[id=review-tab]",
+            run: "click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message button:contains('comment')",
+            run: "click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message .o-mail-Composer-input",
+            run: "edit Putting a comment...",
+        },
+        // When the comment box is closed, the content of the composer is preserved
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message button:contains('comment')",
+            run: "click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message:not(:has(.o-mail-Composer))",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message button:contains('comment')",
+            run: "click",
+        },
+        {
+            trigger:
+                "#chatterRoot:shadow .o-mail-Message .o-mail-Composer-input:value('Putting a comment...')",
+        },
+        // Send the comment
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message a:contains('save')",
+            run: "click",
+        },
+        {
+            trigger:
+                "#chatterRoot:shadow .o-mail-Message-body:contains('Putting a comment...'):not(:has(button:contains('comment')))",
+        },
```

Comment:
Positive assertions on presence of the `comment` button is on `.o-mail-Message`, whereas here the negative is on `.o-mail-Message-body`. This means the negative assertion is not well covered here and might pass while it shouldn't: the button could have been moved outside of body but still inside the message.

Make sure to keep using `.o-mail-Message` like above, so negative assertion is reliable.